### DISCUSSION TITLE
ERROR: configuring ingress for resource {ressource}: the transport property of all bindings should match

### DISCUSSION THREAD

**Discussion by SalomonHenke** [microsoft/aspire#2043] [Q&A] [answered]
Mornings everyone
Trying to deploy via. the azd init but get the following error:
ERROR: configuring ingress for resource messaging: the transport property of all bindings should match

Not sure why. And if this is aspire specific or me thats missing something. 
![image](https://github.com/dotnet/aspire/assets/96744651/31a630f4-0b11-4643-8922-0f44a9ce82d9)

The manifest file looks correct. It is irrespective of me choosing public / private on the configuration in the previous step.
![image](https://github.com/dotnet/aspire/assets/96744651/3280adf4-635f-4068-b779-482e450cb552)

Help is appreciated

**Comment by davidfowl** [comment, marked as answer]
This will be fixed in preview3 https://github.com/dotnet/aspire/pull/1985

**Reply by zbarrier** [reply]
Hi @davidfowl , I get the same error for my Microsoft Orleans Silo Host project.  Is there a fix coming for it or am I doing something wrong?

**Reply by davidfowl** [reply]
The Orleans app won’t deploy until preview5 and a newer azd version.
